test/item_effect_increase_stat.c

```diff
@@ -0,0 +1,22 @@
+#include "global.h"
+#include "test_battle.h"
+
+SINGLE_BATTLE_TEST("X-Attack sharply raises battler's attack stat", s16 damage)
+{
+    u16 useItem;
+    PARAMETRIZE { useItem = FALSE; }
+    PARAMETRIZE { useItem = TRUE; }
+    GIVEN {
+        ASSUME(gItems[ITEM_X_ATTACK].battleUsage == EFFECT_ITEM_INCREASE_STAT);
```

Comment:
Correct me if I'm wrong, but I think possibly this should also `ASSUME(B_X_ITEMS_BUFF >= GEN_7);`?